graphql/data/schema.js

```diff
@@ -2025,6 +2025,7 @@ const setUserBkgDailyImageMutation = mutationWithClientMutationId({
   name: 'SetUserBkgDailyImage',
   inputFields: {
     userId: { type: new GraphQLNonNull(GraphQLString) },
+    //  TODO - remove once we update FE to not send category ID up anymore
```

Comment:
In fact, we should leave this and assume some clients will be outdated (they will be).

Reply:
This is where we'd want to formally deprecate the argument and then remove it in a number of months. (We haven't set a process for this yet.)